list.go

```diff
@@ -121,8 +121,10 @@ func NewExecError(err error, stderr string) *ExecError {
 	return &ExecError{err, stderr}
 }
 
-func GoList(workDir string, path string) ([]Package, error) {
-	cmd := exec.Command("go", "list", "-json", path)
+func GoList(workDir string, path string, args ...string) ([]Package, error) {
+	cmdArgs := append([]string{"list", "-json"}, args...)
+	cmdArgs = append(cmdArgs, path)
+	cmd := exec.Command("go", cmdArgs...)
```

Comment:
It may be helpful to keep all the arguments in one slice and then pass it as `exec.Command(cmdArgs...)` as we can then add log statement such as `"[DEBUG] Executing command %q", cmdArgs` (which we should, at some point, when https://github.com/hashicorp/tf-sdk-migrator/pull/20 is merged) but that's really a nitpick.

Reply:
Great suggestion, thanks

Reply:
Sounds great! The signature of `exec.Command` is:
```
func Command(name string, arg ...string) *Cmd 
```

so I'm going to keep the command name `go` separate now for clarity. Good idea to include the log message.